src/components/marker-chart/Canvas.js

```diff
@@ -550,36 +551,90 @@ class MarkerChartCanvasImpl extends React.PureComponent<Props, State> {
       timelineTrackOrganization,
       viewport: { viewportLeft, viewportRight, viewportTop, containerWidth },
     } = this.props;
-    if (x < marginLeft - MARKER_DOT_RADIUS) {
+
+    // Note: we may want to increase this value to hit markers that are farther.
+    const dotRadius: CssPixels = MARKER_DOT_RADIUS * rowHeight;
+    if (x < marginLeft - dotRadius) {
       return null;
     }
+
     let markerIndex = null;
     let rowIndexOfLabel = null;
     const markerContainerWidth = containerWidth - marginLeft - marginRight;
 
     const rangeLength: Milliseconds = rangeEnd - rangeStart;
+
+    // Reminder: this is a value between 0 and 1, and represents a percentage of
+    // the full time range.
     const viewportLength: UnitIntervalOfProfileRange =
       viewportRight - viewportLeft;
-    const unitIntervalTime: UnitIntervalOfProfileRange =
+
+    // This is the x position in terms of unit interval (so, between 0 and 1).
+    const xInUnitInterval: UnitIntervalOfProfileRange =
```

Comment:
I renamed a few variables to make them more understandable -- for me at least. Please tell me what you think!